keyboards/tiger910/keymaps/via/keymap.c

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2021 Weirdo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum custem_layers {
+    _BASE,
+    _FN,
+    _FN1,
+    _FN2
+};
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT(
+    KC_ESC,   KC_1,    KC_2,    KC_3,   KC_4,   KC_5,    KC_6,    KC_7,   KC_8,    KC_9,     KC_0,     KC_MINS,   KC_EQL,    KC_BSPC,  KC_BSPC,  \
```

Comment:
The backslashes should be removed from the keymap. They're not actually needed (and are a hold over from older versions)

Reply:
OK

Reply:
OK